### ISSUE TITLE
CI failure on python 3.5 caused by ftfy

### ISSUE THREAD

**Issue by fabiocaccamo** [LuminosoInsight/python-ftfy#174]
Hello, 
I'm using `python-ftfy` in [`python-benedict`](https://github.com/fabiocaccamo/python-benedict), but the CI (both travis and github actions) started to fails on python 3.5:
```
File "/opt/hostedtoolcache/Python/3.5.10/x64/lib/python3.5/site-packages/ftfy/__init__.py", line 32
    text: str
        ^
SyntaxError: invalid syntax
```

**Comment by rspeer** [comment]
Okay, hmm. The problem here is that I forgot to specify `python_requires=">=3.6"` in `setup.py`.

Python 3.5 was EOLed in 2020, but when you install ftfy on Python 3.5, what _should_ happen is you get the latest supported version (ftfy 5.9).

I'll work on fixing this and see if I can yank 6.0 so that others on Python 3.5 don't encounter this problem. A quick fix in your case would be to depend on `ftfy < 6`.

**Comment by fabiocaccamo** [comment]
@rspeer thank you very much for the quick feedback and the fix suggestion.

**Comment by rspeer** [comment]
Fixed in 6.0.1.
